File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
const [N, M] = input[0].split(' ').map(Number);
const steps = input.slice(1);
const baskets = Array.from(new Array(N), (_, i) => i + 1);

for (let step of steps) {
    const [i, j] = step.split(' ').map(Number);
    
    [baskets[i - 1], baskets[j - 1]] = [baskets[j - 1], baskets[i - 1]];
}

const answer = baskets.join(' ');

console.log(answer);

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
import sys

N, M = map(int, input().split())
baskets = [i+1 for i in range(N)]

for line in sys.stdin:
    i, j = map(int, line.rstrip().split())
    [baskets[i-1], baskets[j-1]] = [baskets[j-1], baskets[i-1]]

print(' '.join(map(str, baskets)))